src/database/Firebase.py

```diff
@@ -48,34 +49,42 @@ def checkVercode(code: int, id: int) -> bool:
 
 
 async def sendEmail(event: hikari.DMMessageCreateEvent):
+
     # Declare var, FIX NETID/ver_code input parsing
-    email_sender = getenv("email")
-    email_password = getenv("emailpass")
+    email_sender = os.getenv("email")
+    email_password = os.getenv("emailpass")
     email_receiver = event.content + "@scarletmail.rutgers.edu"
     ver_code = (
         db.child("users").child(f"{event.author_id}").child("ver_code").get().val()
     )
     netID = db.child("users").child(f"{event.author_id}").child("netID").get().val()
 
     subject = "Your CoolCat Verification"
-    body = f"""
-    Here is your CoolCat verification code:
-    {ver_code}
-    Please send it back to the bot through its DM!
-    """
 
     # Email formatting, em is the object
     em = EmailMessage()
     em["From"] = email_sender
     em["To"] = email_receiver
     em["Subject"] = subject
-    em.set_content(body)
+    em.set_content("Placeholder, do not remove")
+
+    # Import email template (html) data from github
+    link = "https://raw.githubusercontent.com/rutgersesports/rutgersverifybot/main/src/database/emailtemplate.html"
```

Comment:
I think you should get the HTML code from the codebase itself instead of referencing from a GitHub URL.